src/database/DataTypes/Item.js

```diff
@@ -176,7 +176,7 @@ export class Item extends Realm.Object {
       .filtered("requisition.status == 'finalised'")
       .sorted('requisition.entryDate', true)[0];
 
-    return mostRecentRequisitionItem?.requisition?.confirmDate;
+    return mostRecentRequisitionItem?.requisition?.entryDate;
```

Comment:
I'm stupid:(

Reply:
ah, I would have said `confirmDate` too. Probably did, in the PR review. I see it now.